index.js

```diff
@@ -0,0 +1,259 @@
+const navWrapper = document.querySelector(".nav-wrapper");
+const navToggle = document.querySelector(".mobile-nav-toggle");
+const primaryNavigation = document.querySelector(".primary-navigation");
+
+document.body.addEventListener("click", (e) => {
+  if (
+    !navWrapper.contains(e.target) &&
+    navToggle.attributes[2].nodeValue === "true"
+  ) {
+    closeNav();
+  }
+});
+navWrapper.addEventListener("click", (e) => {
+  if (navToggle.attributes[2].nodeValue === "false") {
+    showNav();
+  } else {
+    closeNav();
+  }
+});
+
+const showNav = () => {
+  navToggle.setAttribute("aria-expanded", true);
+  primaryNavigation.toggleAttribute("opening");
+  primaryNavigation.addEventListener(
+    "animationend",
```

Comment:
Nice job, I've never used this event
https://developer.mozilla.org/en-US/docs/Web/API/Element/animationend_event

Reply:
Yeah, I was looking back at how I wrote the event listener. Would it be a better practice to explicitly state the animation name I'm listening for versus how I have it now? Which is basically looking for "an" animation to finish.

Reply:
Sure